src/iterative_sorting/iterative_sorting.py

```diff
@@ -38,10 +38,15 @@ def bubble_sort( arr ):
 # go through array two at a time
 # compare first element to second element, second to third, etc for outside loop
 # internal: if 1st < 2nd number, keep
-# interla: if 1st > 2nd number, swap positions
-
-    swapped = False
-
+# internal: if 1st > 2nd number, swap positions
+
+    swapped = True
+    while swapped:
+        swapped = False
+        for i in range(0, len(arr) - 1 ): # start at 0 to the length of the array - 1
+            if arr[i] > arr[i + 1]: # if the element of the array is  greater than the element + 1
+                arr[i], arr[i + 1] = arr[i + 1], arr[i]
+                swapped = True
     return arr
 
```

Comment:
great job on the bubble, there is an optimized bubble as well, that bubbles on the way back, something you may want to look into if you have free time